LEAF_Nexus/db_upgrade/Update_OC_DB_202062600-2020082400.sql

```diff
@@ -0,0 +1,10 @@
+START TRANSACTION;
+
+CREATE TABLE `test` (

```

Comment:
?

Reply:
why do we need this db script?

Reply:
This DB script is no longer needed please remove.

Reply:
done